### PR TITLE
HYDRA-807 : Add Unit Test for Maya Display Modes

### PR DESCRIPTION
**Pull request by roopavr-adsk**
Tests Maya Display mode for:
Smooth Shade All
Use default material
wireframe on shaded
Textured
Use all Lights
Shadows

### CODE REVIEW

File: test/lib/mayaUsd/render/mayaToHydra/testMayaDisplayModes.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        #Smoothshaded Material
+        cmds.modelEditor(panel, edit=True, displayAppearance="smoothShaded")
+        cmds.refresh()
+        self.assertSnapshotClose("smoothShaded" + ".png", self.imageDiffFailThreshold, self.imageDiffFailPercent)
+        
+
+    def test_MayaDisplayModes(self):
+        cmds.file(new=True, force=True)
+
+        # Load a maya scene with a maya native sphere, usd sphere and some lights, with HdStorm already being the viewport renderer.
```
Comment (lilike-adsk) [inline test/lib/mayaUsd/render/mayaToHydra/testMayaDisplayModes.py]: does the scene contain usd prim? or this comment needs an update?

**Review by lilike-adsk — COMMENTED** [review]
I guess we can delete the ".mayaSwatches" folder, it seems to be useless

**Review by lilike-adsk — DISMISSED** [review]

File: test/lib/mayaUsd/render/mayaToHydra/testMayaDisplayModes.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        cmds.modelEditor(panel, edit=True, displayTextures=False)
+
+        #Smoothshaded Material
+        cmds.modelEditor(panel, edit=True, displayAppearance="smoothShaded")
+        cmds.refresh()
+        self.assertSnapshotClose("smoothShaded" + ".png", self.imageDiffFailThreshold, self.imageDiffFailPercent)
+        
+
+    def test_MayaDisplayModes(self):
+        cmds.file(new=True, force=True)
```
Comment (debloip-adsk) [inline test/lib/mayaUsd/render/mayaToHydra/testMayaDisplayModes.py:91]: Just noticed this, but as an FYI, this is no longer necessary, as that command will now be run by default before each individual test (see [mtohUtils.py](https://github.com/Autodesk/maya-hydra/blob/dev/test/testUtils/mtohUtils.py#L61-L65)). Shouldn't cause issues though.

**Review by debloip-adsk — APPROVED** [review]
Looks good, thanks! Added a small FYI, I'll let you decide if you want to adjust this test now or merge it as-is

File: test/lib/mayaUsd/render/mayaToHydra/testMayaDisplayModes.py (hunk 2 above)
Comment (roopavr-adsk) [reply test/lib/mayaUsd/render/mayaToHydra/testMayaDisplayModes.py:91]: noted. will incorporate it in my next PR.

**Pull request merged**